README.md

```diff
@@ -1,5 +1,11 @@
 ## wagtail-markdown: Markdown fields and blocks for Wagtail
 
+[![Build status](https://github.com/torchbox/wagtail-markdown/actions/workflows/ci.yml/badge.svg?style=for-the-badge)](https://github.com/torchbox/wagtail-markdown/actions)
```

Comment:
This doesn’t seem to support that `style`.

Reply:
I forgot the shields part

https://img.shields.io/github/workflow/status/torchbox/wagtail-markdown/CI/main?style=for-the-badge
https://img.shields.io/github/workflow/status/torchbox/wagtail-markdown/CI/chore/tidy-ups?style=for-the-badge